common/vpcclient/vpcvolume/get_volume.go

```diff
@@ -69,7 +70,7 @@ func (vs *VolumeService) GetVolumeByName(volumeName string, ctxLogger *zap.Logge
 
 	if volumes != nil {
 		volumeslist := volumes.Volumes
-		if volumeslist != nil && len(volumeslist) > 0 {
+		if volumeslist != nil {
```

Comment:
better to check the length instead of pointer

Reply:
Done